src/lib/tabs/tab.ts

```diff
@@ -45,8 +46,11 @@ export class MatTab extends _MatTabMixinBase implements OnInit, CanDisable, OnCh
   /** Content for the tab label given by <ng-template mat-tab-label>. */
   @ContentChild(MatTabLabel) templateLabel: MatTabLabel;
 
+  /** User provided template that we are going to use instead of implicitContent template */
```

Comment:
"Template provided in the tab content that will be used if present, used to enable lazy-loading"